### PR TITLE
[NUI] Fix CodingStyle error and spellings

### PR DESCRIPTION
**Pull request by dongsug-song**
[NUI] Fix CodingStyle error and spellings

### CODE REVIEW

File: Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs [hunk 1]
```diff
@@ -622,7 +627,7 @@ private VisualView CreateVisualView3()
         {
             VisualView CurrentVisualView = new VisualView();
             CurrentVisualView.Size2D = new Size2D(CurrentWidth, CurrentHeight);
-            CurrentVisualView.ParentOrigin = ParentOrigin.TopLeft;;
+            CurrentVisualView.ParentOrigin = ParentOrigin.TopLeft; ;
```
Comment (AdunFang) [inline Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs:630]: There are two semicolons at the end of this line of code. Is there any special reason?

File: Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs [hunk 2]
```diff
@@ -683,7 +688,7 @@ void Initialize()
             CurrentWidth = WindowWidth - 2 * FrameSize;
             CurrentHeight = (int)((WindowHeight - 3 * FrameSize) * (1 - ButtonToMainRatio));
             MainVisualView.Size2D = new Size2D(3 * WindowWidth, CurrentHeight);
-            MainVisualView.ParentOrigin = ParentOrigin.TopLeft;;
+            MainVisualView.ParentOrigin = ParentOrigin.TopLeft; ;
```
Comment (AdunFang) [inline Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs:691]: There are two semicolons at the end of this line of code. Is there any special reason?

File: Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs [hunk 3]
```diff
@@ -502,7 +503,7 @@ private VisualView CreateVisualView1()
         {
             VisualView CurrentVisualView = new VisualView();
             CurrentVisualView.Size2D = new Size2D(CurrentWidth, CurrentHeight);
-            CurrentVisualView.ParentOrigin = ParentOrigin.TopLeft;;
+            CurrentVisualView.ParentOrigin = ParentOrigin.TopLeft; ;
```
Comment (AdunFang) [inline Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs:506]: There are two semicolons at the end of this line of code. Is there any special reason?

File: Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs [hunk 4]
```diff
@@ -579,7 +580,7 @@ private VisualView CreateVisualView2()
...
```
Comment (AdunFang) [inline Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs:583]: There are two semicolons at the end of this line of code. Is there any special reason?

File: Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs (hunk 3 above)
Comment (dongsug-song) [reply Mobile/NUI/Visuals/NUI_Visuals/NUI_Visuals.cs]: done

**Pull request merged**